src/test/java/jenkins/plugins/git/GitSCMFileSystemTest.java

```diff
@@ -427,6 +429,43 @@ public void filesystem_supports_descriptor() throws Exception {
         assertTrue(SCMFileSystem.supports(descriptor));
     }
 
+    @Issue("JENKINS-42971")
+    @Test
```

Comment:
May also want to assert behavior with a null env, undefined variable, etc.

Reply:
there is one for null env, but this is internal method, and there are many checks already in the calling method